Makefile

```diff
@@ -63,7 +67,7 @@ validate:
 
 .PHONY: autoreview
 autoreview: 	 ## Runs various checks (static analysis & AutoReview test suite)
-autoreview: phpstan validate test-autoreview
+autoreview: check_trailing_whitespaces phpstan validate test-autoreview
```

Comment:
since it does a CS change as opposed to a quality control per se, I think it is more appropriate to do it in `cs` than `autoreview`